### DISCUSSION TITLE
SF5.4 and sensio/framework-extra-bundle

### DISCUSSION THREAD

**Discussion by sdespont** [symfony/symfony#49473] [Q&A] [answered]
Sorry to ask a silly question, but I am a bit confusing about "sensio/framework-extra-bundle". I read that the full features have been merged into Symfony 6.2. So, because I am using Symfony 5.4, the bundle is still needed even if it is not maintained anymore, right?

**Comment by wouterj** [comment, marked as answer]
> So, because I am using Symfony 5.4, the bundle is still needed even if it is not maintained anymore, right?

Yes, indeed.

**Reply by sdespont** [reply]
@wouterj Thank you for your answer
